modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache

```diff
@@ -46,6 +52,14 @@ import java.io.Serializable
 		private const val serialVersionUID: Long = 123
 	}
 {{/serializableModel}}
+{{#discriminator}}
+{{#requiredVars}}
+{{>interface_req_var}}
+{{/requiredVars}}
+{{#optionalVars}}
+{{>interface_opt_var}}
+{{/optionalVars}}
+{{/discriminator}}
```

Comment:
Not tested, but I think this would need to be the following to avoid empty newlines:

```
{{/serializableModel}}{{#discriminator}}{{#requiredVars}}
{{>interface_req_var}}{{/requiredVars}}{{#optionalVars}}
{{>interface_opt_var}}{{/optionalVars}}{{/discriminator}}
```

Reply:
Corrected, thanks!